boards/firefly/Makefile

```diff
@@ -0,0 +1,5 @@
+MODULE = board
+
+DIRS += $(RIOTBOARD)/remote-common
```

Comment:
A large refactoring of the board configuration was recently merged. Please update this file accordingly